### PR TITLE
Add initial description on board of directions

### PR DESCRIPTION
**Pull request by rubensworks**
@pietercolpaert Could you check if you agree with the contents of this new page?

### CODE REVIEW

File: pages/association/board.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+---
+
+The [Comunica Association](/association/) has a Board of Directors
+that makes decisions with respect to the Comunica Association.
+This page describes details on who are the members of this board, what it does, and how it works.
+
+## Members
+
+* [Ruben Taelman](https://www.rubensworks.net/) [(IDLab, Ghent University – imec)](https://knows.idlab.ugent.be/) - Code maintainer and [BDFL](https://en.wikipedia.org/wiki/Benevolent_dictator_for_life)
+* [Pieter Colpaert](https://pietercolpaert.be/) [(IDLab, Ghent University – imec)](https://knows.idlab.ugent.be/) - Management contributor
```
Comment (pietercolpaert) [inline pages/association/board.md:13]: Strategic coordination? Management contributor sounds weird

File: pages/association/board.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+## Becoming a Board Member
+
+There are two ways to become a Board Member:
+
+1. Become a financial contributor via [Open Collective](https://opencollective.com/comunica-association) of at least the Silver tier.
+2. Become a regular [contributor](/contribute/) in any other way, with a dedication of at least two hours per week.
+
+## Decision-making Process
+
+At least once every quarter, the board virtually meets for a board meeting.
```
Comment (pietercolpaert) [inline pages/association/board.md:35]: I’d propose at least once a year. We’ll see how it goes, but you’ll notice once every quarter will be quite a lot to assemble all board members.

File: pages/association/board.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+Every decision must be represented by an issue on a (yet to be created) GitHub repository,
+and describe a concrete action once the issue is accepted. 
+During the meeting, such issues can be voted upon with either a 👍 (=1), 👎 (=-1), or a 👀 (=0),
+and every member can place exactly one vote.
+In case of a tie, the final decision is up to the chair.
+A BDFL may optionally overrule any (final) vote if this person considers this decision to be detrimental to the future of Comunica or the Comunica Association.
+Non-attending members may raise their vote for up to two weeks after the meeting.
+Once a vote is final, an action will be carried out by the executive contributors.
+
+It is expected that minutes are scribed for each meeting by a volunteer,
```
Comment (pietercolpaert) [inline pages/association/board.md:54]: Remove `it is expected that`

File: pages/association/board.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+At least once every quarter, the board virtually meets for a board meeting.
+Not all members are required to be present at each meeting.
+The chair is expected to prepare an agenda ahead of time,
+which should contain points raised by the board members.
+A meeting may be skipped if there are no objections from members.
+
+The chair is appointed by the board members, and may be changed at any time through a decision.
+The title of [BDFL](https://en.wikipedia.org/wiki/Benevolent_dictator_for_life) is reserved for one person,
+and can only be passed on to someone else by the current BDFL.
```
Comment (pietercolpaert) [inline pages/association/board.md:43]: But then it’s not FL any longer? ;-)

Agree with the concept of a BDFL: members give their trust in your decisions. If they don’t like your decisions, they can still fork the code and go elsewhere. I just wonder whether the BDFL name is a bit outdated and harsh/not-friendly, and whether we’d be able to come up with something else? Could we maybe call it something like “code-base curator”? In the naming we should show that we are working to enable their use cases, and not that there is a dictator that may overrule the board’s decision at any time. There may however be good reasons for a code curator to overrule a board’s decision: it should also be in the interest of how the code is currently being set-up.

**Review by pietercolpaert — CHANGES_REQUESTED** [review]

**Comment by rubensworks** [comment]
Thanks! I agree with all comments.

**Pull request merged**